plugins/browser-plugin-media-tracking/src/wgEvents.ts

```diff
@@ -0,0 +1,45 @@
+// https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement#events
```

Comment:
nit: `wgEvents.ts` isn't that obvious what `wg` is. Maybe `mediaEvents.ts` is an ok name?